### PR TITLE
Support multiple networks

### PR DESCRIPTION
**Pull request by democrazyfi**

### CODE REVIEW

File: frontend/app/constants.ts [hunk 1]
```diff
@@ -1,12 +1,5 @@
 export const constants = {
-  UNIVERSAL_GRAVE_FORWARDER: '0x1e2f66d2dC19bB13617D1d06CBb3E8d8aa4567e3',
-  LSP1_URD_VAULT_TESTNET: '0xBc7b3980614215c8090dF310661685Cc393B601A',
   IPFS: 'https://api.universalprofile.cloud/ipfs',
-  LUKSO_EXPLORER: {
-    TESTNET: {
-      ADDRESS: 'https://explorer.execution.testnet.lukso.network/address/',
-    },
-  },
```
Comment (democrazyfi) [inline frontend/app/constants.ts:9]: moved to network config

File: frontend/constants/networks.ts [hunk 2]
```diff
@@ -1,16 +1,46 @@
...
+
+const NETWORKS = {
   mainnet: {
     chainId: 42,
     symbol: 'LYX',
     rpcUrl: process.env.NEXT_PUBLIC_MAINNET_RPC_URL,
     fallbackRpcUrl: 'https://rpc.lukso.gateway.fm',
     explorerURL: 'https://explorer.execution.mainnet.lukso.network',
+    universalGraveForwarder: '',
+    lsp1UrdVault: '',
```
Comment (democrazyfi) [inline frontend/constants/networks.ts:20]: will be added when we deploy final contracts

File: frontend/components/JoinGravePanel.tsx [hunk 3]
```diff
@@ -194,7 +197,7 @@ const JoinGravePanel: React.FC = () => {
                 {step.completeText.text}
                 {step.completeText.address && (
                   <a
-                    href={`${constants.LUKSO_EXPLORER.TESTNET.ADDRESS}${step.completeText.address}`}
+                    href={`${networkConfig.luksoExplorer}${step.completeText.address}`}
```
Comment (tehnalogos) [inline frontend/components/JoinGravePanel.tsx:200]: @democrazyfi there's issue here. `networkConfig.luksoExplorer` does not have `/address/` path that this previously relied on. i think this is the case in a few other cases.

File: frontend/components/wallet/WalletContext.tsx [hunk 4]
```diff
@@ -32,6 +36,7 @@ const defaultImplementation: WalletContextType = {
   addGraveVault: () => {
     // Default addGraveVault implementation
   },
+  connectedChainId: undefined,
```
Comment (Ledespana) [inline frontend/components/wallet/WalletContext.tsx:39]: could it be null instead of undefined?

Comment (democrazyfi) [reply frontend/components/wallet/WalletContext.tsx:39]: As far as I know undefined means a var is declared but not yet assigned a value
null is an explicit assigned value, i.e we're saying "null" would be an acceptable connectedChainId
in this case would undefined be more appropriate? as we have yet to fetch a value (but when we do, it wouldn't be "null")

**Review by tehnalogos — APPROVED** [review]

**Pull request merged**